pkg/fanal/secret/builtin-rules.go

```diff
@@ -76,7 +76,8 @@ const (
 	aws = `(aws)?_?`
 )
 
-var builtinRules = []Rule{
+// This var is exported for trivy-plugin-aqua purposes only
+var BuiltinRules = []Rule{
```

Comment:
You can disable built-in rules and add new rules via `Config`.
https://github.com/aquasecurity/trivy/blob/36fc4aecec27824da21c6324b9f5a1b91fffd565/pkg/fanal/secret/scanner.go#L293

Do you still need to export `BuiltinRules`? It is better to have fewer public functions and variables since it is less likely to break the interface.

Reply:
@VaismanLior is also familiar with the design.

Reply:
I don't intent to change the variable to add rules, I just need the IDs and titles.
I can create a function that extracts this data instead if you prefer

Reply:
Got it. Sounds better since it keeps the variable read-only.